maquetado/scss/detail.scss

```diff
@@ -0,0 +1,58 @@
+%equal-font-heights {
+ 
+  font-family: Roboto;
+  display: flex;
+  align-items:baseline;
+
+}
+
+%equal-color1 {
+
+  color: #828282;
+
+}
+
+%equal-color2 {
+
+  color: #000;
+}
+
+
+
+.titulo-libro {
+  @extend %equal-font-heights;  
+  @extend %equal-color2;
```

Comment:
Even though I root for SASS templates, templates for the sake of it are not really that standard. Just for colors, use the `color` attribute